tools/fonts/@/finale/OFL.txt

```diff
@@ -0,0 +1,99 @@
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Finale”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Maestro”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Broadway”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Engraver”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Jazz”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Ash”.
+Copyright © 2021 MakeMusic, Inc. (https://www.makemusic.com/), with Reserved Font Name “Legacy”.
+
```

Comment:
I'm going to merge this PR, but we should consider naming all license files LICENSE.txt, and ideally we should include references to the original source license. For example, for finale fonts we should include this URL:

https://makemusic.zendesk.com/hc/en-us/articles/1500013053461-MakeMusic-Fonts-and-Licensing-Information

Reply:
@ronyeh I made the reference to the link in the README file